### ISSUE TITLE
No revision found pop up error

### ISSUE THREAD

**Issue by MarioDMC** [gitextensions/gitextensions#7035]
## Current behaviour

Hi, Every time I click over a local or remote branch in the GUI, the "No revision found" pop up appears.

![imagen](https://user-images.githubusercontent.com/46493775/63440684-e9e4ca80-c3f5-11e9-9c30-41d4a38b7700.png)

I need to close the pop up in order to complete another action inside the GUI.

Occurs equally on local and remote branches. 

Any other action like fetch, commit, push... works without problem also commits are retrieved correctly.

## Expected behaviour

When I select any branch, I dont what to keep closing the "No revision found" pop up every time I make click over it.

## Steps to reproduce

I Just make click over a branch

## Screenshots

![imagen](https://user-images.githubusercontent.com/46493775/63441431-06353700-c3f7-11e9-993c-f034bf4cf4b4.png)

## Did this work in previous version of GitExtensions

I reinstalled GitExtensions many times and the error keeps appearing.

## Environment

- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.8.3928.0
- DPI 120dpi (125% scaling)

## Diagnostics

I guess is cause by a bad configuration, can you help me to identify what is wrong with my settings?

**Comment by pmiossec** [comment]
I see that you have the most recent version of Git.
Did you update recently? And did you have the error before the update?

**Comment by MarioDMC** [comment]
> I see that you have the most recent version of Git.
> Did you update recently? And did you have the error before the update, so with an older version?

Yes, I also experienced this error when i was using previous version of git (2.20.0) and I just updated today to 2.23.0.

**Comment by RussKie** [comment]
I can't repro the issue.

Is it happening to all repos or a specific one?
What do you have in Gitcommand log (<kbd>F12</kbd>)?

Please record an animation (e.g. with screen2gif).

**Comment by MarioDMC** [comment]
> I can't repro the issue.
> 
> Is it happening to all repos or a specific one?
> What do you have in Gitcommand log (F12)?
> 
> Please record an animation (e.g. with screen2gif).

It happens on all of the repos.

Here it is my screen recording, hope this will be useful.

![gitextensions_error](https://user-images.githubusercontent.com/46493775/63641267-080a2f00-c671-11e9-9313-dba9663df363.gif)

**Comment by RussKie** [comment]
The git log shows errors - git returns with `-1073740791` instead of `0`.
It looks like you have "show first parents" on, and as a result you don't see branches you trying to get to.

**Comment by MarioDMC** [comment]
> 
> 
> The git log shows errors - git returns with `-1073740791` instead of `0`.
> It looks like you have "show first parents" on, and as a result you don't see branches you trying to get to.

I have disabled the "show first parents" but the message keeps popping. 

Also after clicking multiple times over the branch name I'm able to see the branch's content, but the problem persist. 

Here is an example

![gitextensions_error0](https://user-images.githubusercontent.com/46493775/63644473-87b6ee80-c6af-11e9-8ee1-92742e85782f.gif)

**Comment by RussKie** [comment]
There must be something else in your system that slows git gown... 
Again you can see non-zero return codes for git operations. And when you get the revision the return code is zero.
